### PR TITLE
Updating Readme.md

### PR DESCRIPTION
**Pull request by Boneill3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,12 +23,6 @@ An NCurses-based TUI application for tracking activity over the CAN bus and deco
 
 ***
 
-# Configuration
-
-The default configurations provided by CANOpen Monitor can be found in [canopen_monitor/assets](./canopen_monitor/assets). These are the default assets provided. At runtime these configs are copied to `~/.config/canopen-monitor` where they can be modified and the changes will persist.
-
-***
-
```
Comment (thaotran9) [inline README.md:31]: we don't need json files for the new version.

**Comment by Boneill3** [comment]
Looks good to me! I can't think of anything else to throw on there.

**Review by TheDragonMask — APPROVED** [review]
Looks good to me, submitting my review as "approved"!

**Comment by Sanlinlin** [comment]
It also looks good to me.

**Pull request merged**